src/main/scala/beam/utils/FileUtils.scala

```diff
@@ -96,6 +96,7 @@ object FileUtils extends LazyLogging {
   def downloadFile(source: String, target: String): Unit = {
     assert(source != null)
     assert(target != null)
+    println(s"@@@@@ downloading $source to $target")
```

Comment:
Please, remove `println` and use logging instead

Reply:
wip. missing the label

Reply:
done